themes/default.sh

```diff
@@ -93,6 +102,9 @@ fancygit_theme_builder() {
         prompt_path="${path}${prompt_env} ${prompt_path} ${path_end}${workdir_color_tag}${separator}${none}"
         PS1="${prompt_time}${prompt_user}${prompt_symbol}${prompt_path}${prompt_double_line} "
         return
+    elif [ "HEAD" = "$branch_name" ] && [ "true" = "$show_tag_name" ]
```

Comment:
You can replace the `elif` by an `if`, as it returns one line above.

Reply:
I didn't pay attention! It's ok !